paper-api/src/main/java/org/bukkit/event/entity/EntityPortalEvent.java

```diff
@@ -47,6 +51,7 @@ public EntityPortalEvent(@NotNull Entity entity, @NotNull Location from, @Nullab
         this.searchRadius = searchRadius;
         this.canCreatePortal = canCreatePortal;
         this.creationRadius = creationRadius;
+        this.worldBorder = from.getWorld().getWorldBorder();
```

Comment:
```suggestion
        this.worldBorder = to.getWorld().getWorldBorder();
```

same for PlayerPortalEvent

Reply:
is there a reason you can think of that 'to' is Nullable? Should I change it to NotNull as it is internal API anyways? I don't know if theres a sensible default for the worldborder if the 'to' Location is null

Reply:
The unit tests we inherited are stupid and use null as the to location for some test, I think that that caused the person who added the annotations to consider "to" to be a valid target; That really should be switched at some point